bin/zuul

```diff
@@ -22,6 +22,7 @@ program
 .option('--local [port]', 'port for manual testing in a local browser')
 .option('--tunnel', 'establish a tunnel for outside access. only used when --local is specified')
 .option('--phantom', 'run tests in phantomjs. PhantomJS must be installed separately.')
+.option('--browserstack', 'run tests in BrowserStack instead of Sauce Labs.')
```

Comment:
wondering if `--platform <sauce | browserstack | phantom >` wouldn't be a better flag